keras_maskrcnn/bin/test_csv_generator.py

```diff
@@ -0,0 +1,28 @@
+from keras_maskrcnn.preprocessing.csv_generator import CSVGenerator
```

Comment:
What does this tool do that `debug.py` doesn't do?

Reply:
I just tested a couple of things, it was sort of a unit testing. I can checkthe other one and add a proper unit testing.

Reply:
Yeah I think it's better to remove this and add actual unit testing.